_src/static/sass/base/_typography.scss

```diff
@@ -15,6 +15,31 @@ h1, h2, h3, h4, h5, h6 {
     @extend %reset-headings;
 }
 
+h1 {
```

Comment:
NOTE: placeholders until real headings are in (to text scale/font-size function).